pkgs/development/tools/csvq/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "csvq";
+  version = "1.17.11";
+
+  src = fetchFromGitHub {
+    owner = "mithrandie";
+    repo = "csvq";
+    rev = "v${version}";
+    sha256 = "sha256-cH4uUi/QhOxf9aXh3wRozSKPTd13PnxydKFoG32MIxM=";
```

Comment:
I get a mismatch, I'm guessing this was not updated after the 1.17.1 -> 1.17.11 update.

Reply:
Bumped both the regular and vendored SHAs.